hsc/hsc_consistency.py

```diff
@@ -9,17 +9,35 @@
 from copy import deepcopy
 
 def main():
-    if len(sys.argv)<3:
+
+    fnames = []
+    Lmax = -1
+    savefig = False
+    figsize = (7,7)
+    for argnum in range(1, len(sys.argv)):
```

Comment:
Come on, Adam, just implement this right: 
https://docs.python.org/3/howto/argparse.html

Reply:
My apologies - I am not very familiar with running python scripts in terminal.  I have now made these changes.